collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -2678,7 +2678,11 @@ public void widgetValueChanged(QuestionWidget changedWidget) {
             runOnUiThread(new Runnable() {
                 @Override
                 public void run() {
-                    updateFieldListQuestions(changedWidget.getFormEntryPrompt().getIndex());
+                    try {
+                        updateFieldListQuestions(changedWidget.getFormEntryPrompt().getIndex());
+                    } catch (RuntimeException e) {
+                        Timber.w(e);
```

Comment:
Let's do a `Timber.i` so it doesn't go to Crashlytics. I don't think we can do anything with the information.